test/phx_tools_web/live/phx_tools_landing_live_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule PhxToolsWeb.PhxToolsLandingLiveTest do
+  use PhxToolsWeb.ConnCase, async: true
+
+  import Phoenix.LiveViewTest
+
+  describe "landing page" do
```

Comment:
Add another test case to check whether the proper icon gets different background color.
No need to do 2 tests for both platforms, just pass the user agent that gets detected as Mac for example, and then check the Mac icon gets the background color class added.

Reply:
Still missing this assertion in the test.